install/linux/usr/bin/odemis-select-mic-start

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+: '
+This script is used when there are multiple modular configurations available for METEOR.
+It is recommended to combine it with the exec= command of the Odemis.desktop file.
+However it could be used stand-alone if it is required.
+The syntax to run this script -> odemis-select-mic-start [search_dir] [search_pat].
+The script will search for configuration (.yaml) files with the used search directory
+and pattern. Once files are found microscope names are extracted and placed in a list.
+This list is presented to the user in the form of a (zenity) selection dialog window.
+After selection of the microscope configuration the right yaml file will be used to start Odemis.
+'
+# extract the arguments which were used to start this script and verify the length
+args=( "$@" )
+if test "${#args[@]}" != "2"; then
+        echo "Wrong number of arguments, expected 2, got ${#args[@]}"
+        echo "Syntax is: odemis-select-mic-start [search_dir] [search_pat]"
```

Comment:
I would suggest to rename `search_pat` to `search_ptn` because `pat` is quite similar to `path`. It at least confused me, I was thinking: why is there a search directory and a search path?

Reply:
@sneepstefan is there a reason you did not update this, but resolved the conversation?

Reply:
@tepals  Sorry my bad at the start of my modifications I changed it, then reverted back the whole file eventually and these changes were left out. Added them again.